saas/views/optins.py

```diff
@@ -70,7 +70,7 @@ def get(self, request, *args, **kwargs):
         obj.grant_key = None
         obj.save()
         LOGGER.info("%s accepted role of %s to %s (grant_key=%s)",
-            request.user, obj.role_description, obj.organization,
+            request.user, obj.role_description.title, obj.organization,
```

Comment:
In logs we prefer `slug` and `username`. They are more stable when looking for information during audits.